RefrigeratorSaraFarkash/RefrigeratorSaraFarkash/CItem.cs

```diff
@@ -0,0 +1,124 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace RefrigeratorSaraFarkash
+{
+    internal class CItem:IComparable<CItem>
+    {
+
+        private static int countObj = 0;
+        private int id;
+
+        public int Id
+        {
+            get { return id; }
+            //set { id = countObj; }
+
+        }
+        private string name;
+
+        public string Name
+        {
+            get { return name; }
+            set
+            {
+                if (!value.Equals(""))
+                    name = value;
+                else
+                    throw new ArithmeticException("Illegal name, name must be string");
+            }
+        }
+
+        private int idShelf;
+
+        public int IdShelf
+        {
+            get { return idShelf; }
+            set
+            {
+                if (value >= 0)
+                    idShelf = value;
+                else
+                    throw new ArithmeticException("Illegal idShelf, idShelf must be int");
+            }
+        }
+
+        private string typeI;
```

Comment:
I am guessing you tried to name it type and had a problem with the fact the word type is a saved word in C#.
In this case you should use the full name: itemType (typeItem is grammatically incorrect)

Reply:
fixed